code/game/machinery/doors/firedoor.dm

```diff
@@ -298,7 +301,6 @@
 	opacity = 1
 	explosion_block = 2
 	assemblytype = /obj/structure/firelock_frame/heavy
-	can_force = FALSE
```

Comment:
No longer makes flavour sense, given that we are no longer *literally* forcing the lock open by hand, but using the emergency fallback instead.